src/mlpack/bindings/R/tests/CMakeLists.txt

```
@@ -3,4 +3,4 @@ add_r_binding("" test_r_binding)
 
 add_test(NAME r_binding_test
     COMMAND ${RSCRIPT_EXECUTABLE} -e "library(pkgload); load_all('.'); library(testthat); test_dir('tests/')"
-    WORKING_DIRECTORY ${CMAKE_BINARY_DIR}/src/mlpack/bindings/R/mlpack)
+    WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/src/mlpack/bindings/R/mlpack)
```

```suggestion
    WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/mlpack)
```

I think that this will be correct.  The same changes will need to be applied elsewhere through the binding configuration too.